must-call-checker/src/main/java/org/checkerframework/checker/mustcall/MustCallAnnotatedTypeFactory.java

```diff
@@ -149,22 +164,28 @@ public void addComputedTypeAnnotations(Element elt, AnnotatedTypeMirror type) {
   }
 
   /**
-   * Creates a new @MustCall annotation that is identical to the input, but does not have "close".
+   * Creates a new MustCall annotation that is identical to the input, but does not have "close".
```

Comment:
I personally like the "@" character.  It is a bit more explicit.  In code like the Index Checker that has internal representations with names similar to annotation names, it helps to distinguish between an annotation and the representation of an annotation.

All that said, either way is OK.

Reply:
For this comment and the others in the Must Call Checker, I addressed them in https://github.com/typetools/checker-framework/pull/4484 but not here, since the copy of that checker in this repository will soon be deleted.